extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -105,4 +105,16 @@
      */
     @ConfigItem(defaultValue = "8192")
     public int maximumMessageSize;
+
+    /**
+     * Include message parameters from the log event
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean includeLogMessageParameters;
+
+    /**
+     * Include source code location
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean includeLocation;
```

Comment:
Aren't the default of these values `false`?

Reply:
No, you can check the default values here: https://logging.paluch.biz/examples/jbossas7.html